### PR TITLE
PAINT-282 Cursor

### PR DESCRIPTION
**Pull request by ndrnour**
* As with any tool (e.g. shapes) there shouldn't be radiobuttons in the cursor options, but those buttons.
 * The stroke width number should be with an input field, that it is possible to edit the number easily.

### CODE REVIEW

**Comment by ndrnour** [comment]
test this please

**Comment by ndrnour** [comment]
please run espresso tests

File: Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/dialog/BrushPickerIntegrationTest.java [hunk 1]
```diff
@@ -97,8 +99,8 @@ private void setStrokeWidth(int strokeWidth, int expectedStrokeWidth) {
 		onView(withId(R.id.stroke_width_seek_bar))
 				.perform(setProgress(strokeWidth))
 				.check(matches(withProgress(expectedStrokeWidth)));
-		onView(withId(R.id.stroke_width_width_text))
-				.check(matches(withText(Integer.toString(expectedStrokeWidth))));
+			//onView(withId(R.id.stroke_width_width_text))
```
Comment (ThomasSchwengler) [inline Paintroid/src/androidTest/java/org/catrobat/paintroid/test/espresso/dialog/BrushPickerIntegrationTest.java]: please remove commented code

File: Paintroid/src/main/java/org/catrobat/paintroid/listener/BrushPickerView.java [hunk 2]
```diff
@@ -1,37 +1,43 @@
 /**
- *  Paintroid: An image manipulation application for Android.
- *  Copyright (C) 2010-2015 The Catrobat Team
- *  (<http://developer.catrobat.org/credits>)
+ * Paintroid: An image manipulation application for Android.
```
Comment (ThomasSchwengler) [inline Paintroid/src/main/java/org/catrobat/paintroid/listener/BrushPickerView.java]: please undo changes to the copyright header

File: Paintroid/src/main/java/org/catrobat/paintroid/ui/tools/CustomInputFilter.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+	@Override
+	public CharSequence filter(CharSequence charSequence, int i, int i1, Spanned spanned, int i2, int i3) {
+		try {
+			int input = Integer.parseInt(spanned.toString() + charSequence.toString());
+			if (input <= max && input >= min) {
+				return null;
+			}
+		} catch (NumberFormatException nfe) {
+			Log.e(CustomInputFilter.class.getSimpleName(), nfe.getLocalizedMessage());
```
Comment (ThomasSchwengler) [inline Paintroid/src/main/java/org/catrobat/paintroid/ui/tools/CustomInputFilter.java]: please extract `CustomInputFilter.class.getSimpleName()` to a static final tag

File: Paintroid/src/main/java/org/catrobat/paintroid/listener/BrushPickerView.java [hunk 4]
```diff
@@ -76,6 +83,32 @@ public void colorChanged(int color) {
...
+			}
+
+			@Override
+			public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+			}
+
+			@Override
+			public void afterTextChanged(Editable editable) {
+				String sizeText = brushSizeText.getText().toString();
+				int value = sizeText.isEmpty() ? MIN_BRUSH_SIZE : Integer.parseInt(sizeText);
```
Comment (ThomasSchwengler) [inline Paintroid/src/main/java/org/catrobat/paintroid/listener/BrushPickerView.java]: please catch `NumberFormatException`

**Pull request merged**